src/BuildAfterApp/BuildAfterApp.csproj

```diff
@@ -0,0 +1,79 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!--
+    Forward targets to @(RequiresDelayedBuild) projects after App.Ref has built. Intended for projects that
+    cannot restore without our App.Ref layout and aren't referenced elsewhere in the repo. For projects
+    referenced elsewhere, use something else e.g. @(TestAssetProjectReference) items and
+    FunctionalTestWithAssets.targets for test asset projects that need special handling.
+  -->
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
```

Comment:
Did not bother to handle multiple TFMs because (a) `Publish` is the main target that operates only for a specific TFM and (b) all `@(RequiresDelayedBuild)` projects currently target this TFM.